src/Chatkit.php

```diff
@@ -1382,10 +1488,10 @@ function($curl, $header) use (&$headers)
 ];
 
 const MESSAGE_ID = [ 'message_id' =>
-                        [ 'type' => 'integer',
-                          'missing_message' =>
-                          'You must provide the message ID'
-                        ]
+                     [ 'type' => 'int',
+                       'missing_message' =>
+                       'You must provide the ID of the message to edit'
```

Comment:
This is being used by `fetchMultipartMessage` as well. Either the error message should remain generic, or we have two separate ones.

Reply:
Yep. Bad merge.

Reply:
9c3962c